reminders/tests.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+from django.utils import timezone
+from . import models
+from datetime import datetime
+from events.models import EventParticipant
+from events.tests import new_event, user0  # noqa: F401
+from events.tests import event_participant_creator as participant0  # noqa: F401
+from django.core.exceptions import ValidationError
+from .models import Reminder, time_format
+
+METHOD_0 = models.Reminder.EMethods.EMAIL
+MESSAGES_0 = 'This is a test text.'
+TIME_0 = datetime(2023, 3, 24, 12, 12, 12, 0, tzinfo=timezone.utc)
+
+PAST_DATE_TIME_ERROR = "{} is not a valid date_time"
+EXIST_REMINDER_ERROR = 'reminder already exists'
+
+
+@pytest.fixture
+def reminder_0(participant0):  # noqa: F811
+    return models.Reminder(participant_id=participant0, method=METHOD_0, messages=MESSAGES_0, date_time=TIME_0)
+
+
+def test_new_reminder_0(reminder_0, participant0):  # noqa: F811
+    assert reminder_0.participant_id == participant0
+    assert reminder_0.method == METHOD_0
+    assert reminder_0.messages == MESSAGES_0
+    assert reminder_0.date_time == TIME_0
+
+
+@pytest.fixture
+@pytest.mark.django_db()
+def persist_reminder(reminder_0):
+    reminder_0.participant_id.user_id.save()
+    reminder_0.participant_id.event_id.save()
+    reminder_0.participant_id.save()
+    reminder_0.save()
+    return reminder_0
```

Comment:
instead of repeating this code twice here and in lines # 73-76
you can create a method like save_reminder(reminder) then use it here and in lines # 73-76
```suggestion
    return save_reminder(reminder_0)

def save_reminder(reminder):
    reminder.participant_id.user_id.save()
    reminder.participant_id.event_id.save()
    reminder.participant_id.save()
    reminder.save()
    return reminder

```